README.md

```diff
@@ -405,9 +433,13 @@ async () => {
 }
 ```
 
-#### <a name="consuming-messages-each-batch"></a> eachBatch
+### <a name="consuming-messages-each-batch"></a> eachBatch
 
-Some use cases can be optimized by dealing with batches rather than single messages. This handler will feed your function batches and some utility functions to give your code more flexibility. Be aware that using `eachBatch` is considered a more advanced use case since you will have to understand how session timeouts and heartbeats are connected. All resolved offsets will be automatically committed after the function is executed.
+In order to process huge volumes of messages in a responsive manner, you need consider the `eachBatch` API. Dealing with batches rather than single messages reduces the network traffic and the communication overhead with the broker, allowing your consumer group to eat away at your partition lag in orders of magnitudes faster than `eachMessage`.
```

Comment:
This isn't really true though. `eachMessage` is essentially a default implementation of `eachBatch`. We still consume a batch at a time from the brokers, and the offset is only resolved, not committed, on each message.

Situations where you might want to use `eachBatch` is where you need to do something with all messages at once. For example, maybe you want to send some data from each message to a remote batch API, so that for N messages you call the remote API once.